examples/exhibit-webpack/app.js

```diff
@@ -14,12 +15,27 @@ class Root extends Component {
     viewport: {
       latitude: 37.785164,
       longitude: -122.41669,
-      zoom: 16.140440,
-      bearing: -20.55991,
-      pitch: 60,
-    },
-    width: 500,
-    height: 500,
+      zoom: 14,
+      bearing: 0,
+      pitch: 0,
+      width: 500,
+      height: 500
+    }
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this._resize);
+    this._resize();
+  }
+
+  _resize = () => {
```

Comment:
I personally think we should avoid non-es6 code usage and keep the exhibits as basic as possible